src/api/youtube.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+const API_KEY = 'AIzaSyAN8BNOmhJQ8d6jPsYEx2SXDBnKusT7eNY';
+
+export default axios.create({
```

Comment:
This is working fine but what if you get this functionality working through a custom hook useful for every component that needs to retrieve youtube data